### PR TITLE
[NL interface] Make e2e working with UI pages and updated config format

### PR DESCRIPTION
**Pull request by shifucun**
This needs more work for css and display.

image

### CODE REVIEW

**Comment by shifucun** [comment]
@chejennifer The new folder layout with components and apps makes it very easy to add new page. Good work!

File: static/js/apps/nl_interface/app.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  /**
+   * Config of the page
+   */
+  pageConfig: SubjectPageConfig;
+}
+
+export function App(props: AppPropType): JSX.Element {
+  return (
+    <div id="dc-nl-interface">
+      <h1>Data Commons NL Interface</h1>
```
Comment (pradh) [inline static/js/apps/nl_interface/app.tsx:41]: how about: "Ask Data Commons", "Chart Search", ...

**Review by jehangiramjad — APPROVED** [review]
Very cool! I don't know too much about css so approving for the rest :)

File: server/routes/nl_interface.py [hunk 2]
```diff
@@ -343,7 +343,9 @@ def page(dcid):
   chart_config = _chart_config(place_dcid, main_place_type, main_place_name,
                                child_places_type, highlight_svs, sv2name,
                                peer_buckets)
-
-  print(chart_config)
-
-  return render_template('/nl.html')
+  message = ParseDict(chart_config, subject_page_pb2.SubjectPageConfig())
```
Comment (pradh) [inline server/routes/nl_interface.py:346]: maybe I'm too greedy :)  is it a trivial change to pass in a query param in place of hardcoding in line 322?  then we can test out different queries

**Review by pradh — APPROVED** [review]
This is really cool!!

File: server/routes/nl_interface.py (hunk 2 above)
Comment (shifucun) [reply server/routes/nl_interface.py:345]: Yea, added a query param

File: static/js/apps/nl_interface/app.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/apps/nl_interface/app.tsx]: Done

**Pull request merged**